homeassistant/components/workday/config_flow.py

```diff
@@ -0,0 +1,259 @@
+"""Adds config flow for Workday integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import holidays
+from holidays import HolidayBase
+import voluptuous as vol
+
+from homeassistant.config_entries import (
+    ConfigEntry,
+    ConfigFlow,
+    OptionsFlowWithConfigEntry,
+)
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.selector import (
+    NumberSelector,
+    NumberSelectorConfig,
+    NumberSelectorMode,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+    TextSelector,
+)
+from homeassistant.util import dt
+
+from .const import (
+    ALLOWED_DAYS,
+    CONF_ADD_HOLIDAYS,
+    CONF_COUNTRY,
+    CONF_EXCLUDES,
+    CONF_OFFSET,
+    CONF_PROVINCE,
+    CONF_REMOVE_HOLIDAYS,
+    CONF_WORKDAYS,
+    DEFAULT_EXCLUDES,
+    DEFAULT_NAME,
+    DEFAULT_OFFSET,
+    DEFAULT_WORKDAYS,
+    DOMAIN,
+)
+
+
+def get_providence_to_schema(
+    schema: vol.Schema,
+    options: dict[str, Any],
+) -> vol.Schema:
+    """Update schema with providence from country."""
+    year: int = dt.now().year
+    obj_holidays: HolidayBase = getattr(holidays, options[CONF_COUNTRY])(years=year)
+    new_schema = schema
+
+    if hasattr(obj_holidays, "subdivisions"):
+        div_list = obj_holidays.subdivisions
+        div_list.insert(0, "None")
```

Comment:
I don't like this `"None"` string. I think it should be somehow handled differently, with an empty string.